bubblewrap.c

```diff
@@ -2162,7 +2168,7 @@ main (int    argc,
 
   __debug__ (("forking for child\n"));
 
-  if (opt_unshare_pid || lock_files != NULL || opt_sync_fd != -1)
+  if (!no_reaper && (opt_unshare_pid || lock_files != NULL || opt_sync_fd != -1))
```

Comment:
I think we should also check if the user specified any lock files or a sync-fd and error out, because those require support from our own pid1.